tests/qt/qt.pro

```diff
@@ -1,3 +1,4 @@
 TEMPLATE = subdirs
 
-SUBDIRS += qmetaobject qmetaproperty qobject qmetamethod
+unix:SUBDIRS += qmetaobject qmetaproperty qobject qmetamethod
+win32:SUBDIRS += qmetaproperty
```

Comment:
One should use msvc  instead of win32/unix
Because mingw should be working.
Is there a way to test mingw btw?

```
# Test that have object with too many methods are disabled on MSVC for now because  it reaches the template instentiation limit
!msvc:SUBDIRS += qmetaobject qobject qmetamethod
SUBDIRS += qmetaproperty
```

Reply:
I originally had mingw64: https://github.com/rakhimov/verdigris/commit/d4f3fb6da889b1c632429713ac0b8b256e964c33
Maintaining both with appveyor currently seems PITA. It is for another PR in future maybe.